drivers/hmis/app/models/hmis/role.rb

```diff
@@ -384,6 +384,22 @@ def self.permissions_with_descriptions
           'Administrative',
         ],
       },
+      can_view_client_alerts: {
+        description: 'Grants the ability to view Client Alerts',
+        administrative: true,
+        access: [:editable],
+        categories: [
+          'Administrative',
```

Comment:
Lets put both of these in `Client Access` category instead. I wouldn't say they are administrative

Reply:
Or `Client` if that's a category

Reply:
In this file I only see `Client Access` and `Client Details` -- I'll change it to `Client Details unless you suggest otherwise!